jax/experimental/jax2tf/tests/primitives_test.py

```diff
@@ -383,6 +383,38 @@ def test_population_count(self, harness: primitive_harness.Harness):
     self.ConvertAndCompare(harness.dyn_fun, *harness.dyn_args_maker(self.rng()),
                            expect_tf_exceptions=True)
 
+  @primitive_harness.parameterized(primitive_harness.lax_add_mul)
+  def test_add_mul(self, harness: primitive_harness.Harness):
+    expect_tf_exceptions = False
+    dtype = harness.params["dtype"]
+    f_name = harness.params["f_jax"].__name__
+
+    if not dtype in [dtypes.bfloat16, np.float16, np.float32, np.float64, np.uint8,
+                     np.int8, np.uint16, np.int16, np.int32, np.int64, np.complex64,
+                     np.complex128]:
+      # TODO(bchetioui): tf.math.multiply is only defined for the above types.
```

Comment:
It is a little hard to even tell what types are not defined? Is it uint32 and uint64? I wonder if it would be better to just list those undefined types. Then the test is easier to read if what you are looking for are the undefined cases.

Reply:
Addressed in `bfc8b789`